### ISSUE TITLE
where is log func in loss function ? 

### ISSUE THREAD

**Issue by ddongk** [leftthomas/SRGAN#28]
i think there is no log function in ur loss funcion

right? 
https://github.com/leftthomas/SRGAN/blob/e1f9fdb7dbbb7a023ddbd19340e82101ae9147fa/train.py#L75

**Comment by leftthomas** [comment]
@ddongk  log is unstable, as many papers suggested, we don't computer `log` of Adversarial Loss

**Comment by ddongk** [comment]
@leftthomas  i`m really thanks for ur reply ! could i know why log function is unstable..?

**Comment by ddongk** [comment]
and..do u experimented with log function? is it really not good ?

**Comment by leftthomas** [comment]
@leftthomas you could search it on Google, and `PyTorch` document states `NLL Loss` and `Cross Entropy Loss` why it suggest add `log softmax`
